example/test_problem/Hydro/CCSN/Input__Flag_User.CoreCollapse

```diff
@@ -0,0 +1,10 @@
+# Level                Density
```

Comment:
Change the header `Density` to, for example, `Dummy`. Same for other `Input__Flag_User.*` files.

Reply:
I have updated the headers.